client/src/Movies/Movie.js

```diff
@@ -25,43 +27,27 @@ export default class Movie extends Component {
         console.error(error);
       });
   };
-  // Uncomment this code when you're ready for the stretch problems
-  // componentWillReceiveProps(newProps){
-  //   if(this.props.match.params.id !== newProps.match.params.id){
-  //     this.fetchMovie(newProps.match.params.id);
-  //   }
-  // }
+  //Uncomment this code when you're ready for the stretch problems
+  componentWillReceiveProps(newProps){
```

Comment:
Just for your edification, it's a mistake that this is still in the repo.  The lifecycle method is deprecated and componentDidUpdate is now highly recommended.